atomic_reactor/plugins/post_pulp_pull.py

```diff
@@ -84,4 +84,9 @@ def run(self):
                        image_id)
         self.workflow.builder.image_id = image_id
 
-        return image_id
+        media_types = ['application/json']
```

Comment:
Thinking more about this, I think the answer right now is to make "application/json" conditional on whether pulp_push ran (in the same build as it is running).

Later, to support pulp_publish in the orchestrator build, we'll need pulp_push to store annotations in the worker build that say what v1 content was uploaded, and when we get to that point we can use those annotations to decide whether to include "application/json" in the list.